ScoutSuite/providers/gcp/facade/cloudsql.py

```diff
@@ -0,0 +1,27 @@
+from ScoutSuite.providers.gcp.facade.base import GCPBaseFacade
+from ScoutSuite.providers.gcp.facade.utils import GCPFacadeUtils
+from ScoutSuite.providers.utils import run_concurrently
+
+class CloudSQLFacade(GCPBaseFacade):
+    def __init__(self):
+        super(CloudSQLFacade, self).__init__('sqladmin', 'v1beta4')
+
+    async def get_backups(self, project_id: str, instance_name: str):
+        cloudsql_client = self._get_client()
+        backups_group = cloudsql_client.backupRuns()
+        request = backups_group.list(project=project_id, instance=instance_name)
+        return await GCPFacadeUtils.get_all('items', request, backups_group)
```

Comment:
Shouldn't this be parallelized?

Reply:
It's already parallelized in the `get_all` method.

Reply:
Oh my bad, I looked at the `_get_all`.